### PR TITLE
From  create-melange-app to cma_configuration

### PR DESCRIPTION
**Pull request by JoaoAlexNunes**
i change the name of the file but couldn't create the component folder with the Configuration inside

### CODE REVIEW

File: src/templates/extensions/app/react_re/components/Configuration.re [hunk 1]
```diff
@@ -1,13 +1,4 @@
-/*
- * In ReasonML, the `open` statement is used to bring modules into the current
- * scope, similar to using `import` in JavaScript or TypeScript. However, unlike
- * `import`, `open` makes all of the module's contents immediately available
- * without the need to prefix them with the module name. This is akin to doing a
- * wildcard import in JavaScript:
- *
- * `import * as Create_melange_app from 'Create_melange_app';`
- */
```
Comment (dmmulroy) [inline src/templates/extensions/app/react_re/components/Configuration.re:9]: Lets add this comment back in with the updated package name

File: src/templates/extensions/app/react_re/components/Configuration.re [hunk 2]
```diff
@@ -97,3 +88,4 @@ let make = (~configuration: Configuration.t) => {
     </ul>
   </div>;
 };
+
```
Comment (dmmulroy) [inline src/templates/extensions/app/react_re/components/Configuration.re:91]: nit: Extra space

**Review by dmmulroy — CHANGES_REQUESTED** [review]
Looks super good! I'm pulling now to run through generating some templates to validate. Just want to add a comment back in and fix some formatting 🙏

File: src/templates/extensions/app/react_re/components/Configuration.re (hunk 1 above)
Comment (JoaoAlexNunes) [reply src/templates/extensions/app/react_re/components/Configuration.re:9]: do you want me to add that then?

Comment (dmmulroy) [reply src/templates/extensions/app/react_re/components/Configuration.re:9]: Yes please

Comment (JoaoAlexNunes) [reply src/templates/extensions/app/react_re/components/Configuration.re]: Done

File: src/templates/extensions/app/react_re/components/Configuration.re (hunk 2 above)
Comment (JoaoAlexNunes) [reply src/templates/extensions/app/react_re/components/Configuration.re]: I think is fixed now with the last two push

**Pull request merged**